src/app/users/api/getUserByEmail.js

```diff
@@ -0,0 +1,21 @@
+const { getUserByUsername, getUserByEmail } = require('./../../../infrastructure/oldSecureAccess');
+const logger = require('./../../../infrastructure/logger');
+
+const getSAUser = async (req, res) => {
+  const id = req.params.id;
+  try {
+    let user = await getUserByUsername(id);
+    if (!user) {
+      user = await getUserByEmail(id);
+    }
+    if (!user) {
+      return res.status(404).send();
+    }
+    return res.status(200).send(user);
```

Comment:
should use safeUser when sending user object back.

Example: https://github.com/DFE-Digital/login.dfe.osa-api/blob/master/src/app/authenticate/api/authenticate.js#L18